Source/Core/Core/DSP/Jit/DSPEmitterBase.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/DSP/Jit/DSPEmitterBase.h"
+
+#if defined(_M_X86) || defined(_M_X86_64)
+#include "Core/DSP/Jit/x64/DSPEmitter.h"
+#endif
+
+namespace DSP::JIT
+{
+DSPEmitter::~DSPEmitter() = default;
+
+std::unique_ptr<DSPEmitter> CreateDSPEmitter()
+{
+#if defined(_M_X86) || defined(_M_X86_64)
+  return std::make_unique<x64::DSPEmitter>();
```

Comment:
Shouldn't here be the option if we want to use the DSP recompiler of the DSP interpreter?

Also the #else macro isn't required.

Reply:
> Shouldn't here be the option if we want to use the DSP recompiler of the DSP interpreter?

I don't get what you mean by this

Reply:
I'm missing the
switch(opts.core_type)
{
#if X64
case DSPInitOptions::CoreType::JIT64: return std::make_unique<x64::DSPEmitter>();
#endif
case: interpreter: ...
default: std::make_unique<DSPEmitterNull>();

Reply:
I don't get why the interpreter case needs to be handled here, it's not an emitter type (it's not even a class).